src/char/char.c

```diff
@@ -2747,17 +2748,34 @@ bool char_config_read(const char* cfgName, bool normal){
 #else
 		} else if (strcmpi(w1, "start_point_pre") == 0) {
 #endif
-			char map[MAP_NAME_LENGTH_EXT];
-			short x, y;
-			if (sscanf(w2, "%15[^,],%6hd,%6hd", map, &x, &y) < 3){
-				ShowWarning( "Specified start_point has an invalid format.\n" );
-				continue;
+			int i = 0, fields_length = 3 + 1;
+			char *lineitem, **fields;
+
+			fields = (char**)aMalloc(fields_length * sizeof(char*));
+			lineitem = strtok(w2, ":");
+
+			while (lineitem != NULL) {
```

Comment:
a sub_function for parsing this would be better for clarity. we don't want to clobber char_config_read, with parsing code